packages/server/lib/automation/util.ts

```diff
@@ -12,16 +12,16 @@ export type CyCookie = Pick<chrome.cookies.Cookie, 'name' | 'value' | 'expiratio
 // @ts-ignore
 export type CyCookieFilter = chrome.cookies.GetAllDetails
 
-export const cookieMatches = (cookie: CyCookie | playwright.Cookie, filter: CyCookieFilter) => {
-  if (filter.domain && !domainMatch(filter.domain, cookie.domain)) {
+export const cookieMatches = (cookie: CyCookie | playwright.Cookie, filter?: CyCookieFilter) => {
+  if (filter?.domain && !domainMatch(filter?.domain, cookie.domain)) {
     return false
   }
 
-  if (filter.path && filter.path !== cookie.path) {
+  if (filter?.path && !pathMatch(filter.path, cookie.path)) {
```

Comment:
Is this a part of the migration? Or a separate improvement? Mostly just curious.